examples/chunking/app.ddlog

```diff
@@ -18,13 +24,16 @@ word_features(
 
 tag?(word_id bigint) Categorical(13).
 
-function ext_training
-  over rows like words_raw
-  returns rows like words
-  implementation "udf/ext_training.py" handles tsv lines.
-
-words +=
-  ext_training(wid, word, pos, tag) :- words_raw(wid, word, pos, tag).
+words(sent_id, word_id, word, pos, true_tag, tag_id) :-
+  words_raw(sent_id, word_id, word, pos, true_tag),
+  tags(tag_id, tag),
+  if true_tag = "B-UCP" then ""
+  else if true_tag = "I-UCP" then ""
```

Comment:
I see.  I'm getting the following numbers on my side after a little bit of edits.  Do they look good to you?

```
2016-02-09 18:40:42.179312 ################################################
2016-02-09 18:40:42.179337 # nvar               : 270052
2016-02-09 18:40:42.179360 # nfac               : 1049594
2016-02-09 18:40:42.179392 # nweight            : 268970
2016-02-09 18:40:42.179413 # nedge              : 1354720
2016-02-09 18:40:42.179434 ################################################
2016-02-09 18:40:42.350998 LOADED VARIABLES: #270052
2016-02-09 18:40:42.351067          N_QUERY: #49389
2016-02-09 18:40:42.351087          N_EVID : #220663
2016-02-09 18:40:42.412293 LOADED WEIGHTS: #268970
2016-02-09 18:40:43.577835 LOADED FACTORS: #1049594
```